rollup.config.js

```diff
@@ -1,11 +1,7 @@
-/* @flow strict */
-
-import babel from 'rollup-plugin-babel'
-
-const pkg = require('./package.json')
-
+import typescript from 'rollup-plugin-typescript2'
```

Comment:
[Apparently `typescript2` is slow](https://github.com/rollup/plugins/issues/61#issuecomment-545410511) but I don't mind it here. If we do mind it we could [use `tsc --emitDeclarationOnly` to emit a declaration file](https://github.com/rollup/plugins/issues/61#issuecomment-546389800).

Reply:
I am hoping that `rollup-plugin-typescript2` will down the line be improved to only produce declaration for the output file. `tsc` would not know about the final output in that case.